### PR TITLE
Event exporter

### PR DESCRIPTION
**Pull request by clyang82**
ref: https://issues.redhat.com/browse/ACM-4099

### CODE REVIEW

File: pkg/testdata/event/kube-event-exporter-bad-config.yaml
Comment (yanmxa) [inline pkg/testdata/event/kube-event-exporter-bad-config.yaml:1]: Why do we need this bad event exporter config?

Comment (clyang82) [reply pkg/testdata/event/kube-event-exporter-bad-config.yaml:1]: for UT purpose, refer to https://github.com/stolostron/multicluster-global-hub/blob/def309a0494131094307ea5399f029b5e086f94f/agent/cmd/agent/main_test.go#L353

**Pull request merged**